src/collections/_documentation/development/sdk-dev/event-payloads/breadcrumbs.md

```diff
@@ -3,14 +3,16 @@ title: Breadcrumbs Interface
 sidebar_order: 6
 ---
 
-The breadcrumbs interface specifies a series of application events, or
-`breadcrumbs`, that occurred before the main event. Its canonical name is
-`breadcrumbs`.
+The Breadcrumbs Interface specifies a series of application events, or
+"breadcrumbs", that occurred before and led up to an event.
 
-This interface is an object with a single `values` key containing an ordered
-list of breadcrumb objects. The entries are ordered from oldest to newest.
-Consequently, the last entry in the array should be the last entry before the
-event occurred.
+An [event]({%- link _documentation/development/sdk-dev/event-payloads/index.md
+-%}) may contain one or more breadcrumbs in an attribute named `breadcrumbs`.
+
+This interface is an object with a single `values` attribute containing an
+ordered list of breadcrumb objects. The entries are ordered from oldest to
+newest. Consequently, the last entry in the list should be the last entry before
```

Comment:
```suggestion
 Consequently, the last entry in the list should be the last entry before
```

Reply:
@rhcarvalho I wasn't trying to force a line break. Wanted to delete "newest" because I added it to the previous line.

Reply:
Ah, as far as rendering goes, those two render the same:

```md
A long line with a line
break. Another sentence.
```

```md
A long line with a line break.
Another sentence.
```

The result is a single `<p>` HTML tag displaying all in one line/paragraph:

```
A long line with a line break. Another sentence.
```

When editing markdown it is customary to wrap long lines. To start a new paragraph we need a blank line between blocks of text, and to insert a line break (`<br>` HTML tag) we need to end a line with two (!) (or more) spaces.

https://daringfireball.net/projects/markdown/syntax#p

Sorry if I'm preaching to the priest, probably you know all that. 😊

Reply:
All good. 👍